avm/res/network/network-manager/connectivity-configuration/main.bicep

```
@@ -25,7 +25,7 @@ param appliesToGroups array = []
 param connectivityTopology string
 
 @sys.description('Conditional. List of hub items. This will create peerings between the specified hub and the virtual networks in the network group specified. Required if connectivityTopology is of type "HubAndSpoke".')
-param hubs array = []
+param hubs hubsType?
```

```suggestion
param hubs hubsType
```

You don't need to make both the parameter and type nullable ;)